frontend/server/src/test/resources/snapshots/snapshot4.cfg

```diff
@@ -6,7 +6,7 @@ enable_envvars_config=true
 load_models=noop.mar
 private_key_file=src/test/resources/key.pem
 snapshot_store=FS
-NUM_WORKERS=4
+default_workers_per_init_model=4
```

Comment:
Can we just call this `default_workers_per_model`?

Reply:
done